payment/order.go

```diff
@@ -39,6 +39,27 @@ type OrderItem struct {
 	Description datastore.NullString `json:"description" db:"description"`
 }
 
+// IsValidSKU checks to see if the token provided is one that we've previously created
+func IsValidSKU(sku string) bool {
```

Comment:
Could you update this so that Dev/Staging SKUs are not accepted in Prod.  Otherwise LGTM!

Reply:
Good thinking - https://github.com/brave-intl/bat-go/pull/363/commits/8f2de5081a9475009b144bf773e3a61222446418